src/main/resources/reference.conf

```diff
@@ -27,11 +29,13 @@ akka {
 
     serialization-identifiers {
       "it.pagopa.interop.agreementmanagement.model.persistence.serializer.AgreementAddedSerializer" = 100000
+      "it.pagopa.interop.agreementmanagement.model.persistence.serializer.AgreementDeletedSerializer" = 100001
```

Comment:
are we sure we are not using an old event id?

Reply:
This code refers to VerifiedAttributeUpdatedSerializer, removed on 27/7/2021.

Reply:
ok, then we should not have any regression issue 👍